recipes/osx-sysroot/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "macosx-sysroot" %}
+{% set version = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+build:
+  noarch: generic
+  number: 0
+
+outputs:
+  - name: sysroot_osx-64
+    version: {{ sdk_version }}
+    run_exports:
+      - __osx >={{ sdk_version }}
```

Comment:
linux sysroot [uses](https://github.com/conda-forge/linux-sysroot-feedstock/blob/main/recipe/meta.yaml#L155-L156) a strong run-export here, might be slightly safer?